src/Preg.php

```diff
@@ -180,10 +190,45 @@ private static function removeUtf8Modifier($pattern)
             return '';
         }
 
-        $delimiter = substr($pattern, 0, 1);
+        $delimiter = $pattern[0];
 
         $endDelimiterPosition = strrpos($pattern, $delimiter);
 
         return substr($pattern, 0, $endDelimiterPosition).str_replace('u', '', substr($pattern, $endDelimiterPosition));
     }
+
+    /**
+     * @param string[] $patterns
+     *
+     * @return string[]
+     */
+    private static function verifyPatterns(array $patterns)
+    {
+        return \array_map('PhpCsFixer\Preg::verify', $patterns);
```

Comment:
do not call method by string names.
one will rename method name and this reference will not be picked up by static code analysis/refactor tool

Reply:
@keradus You only say what not to do, but you don't give an example what should be used instead. This is not helpful for me. How is it possible to not  use a string for the method name?

Reply:
sorry, i remembered this topic was raised few times already and wrongly assumed that it was only oversight here.

```
grep -r array_ src -A5
src/Console/ConfigurationResolver.php:        $paths = array_filter(array_map(
src/Console/ConfigurationResolver.php-            static function ($path) {
src/Console/ConfigurationResolver.php-                return realpath($path);
src/Console/ConfigurationResolver.php-            },
src/Console/ConfigurationResolver.php-            $this->getPath()
src/Console/ConfigurationResolver.php-        ));
```

instead of `$paths = array_filter("realpath", $this->getPath());`

Reply:
Alright, I see. This is easy to change now for me. Very helpful, thanks!

Reply:
I update the PR which addresses this. Unfortunately I've ready your comment too late about squashing. Next changes will be separated.